marblerun/reference/coordinator.md

```diff
@@ -30,30 +30,27 @@ For errors, `data` will always be `null`, and `message` contains the specific er
 
 ## Endpoints
 
-The API currently contains the following endpoints. If an endpoint specifies *Returns* for either HTTP GET or HTTP POST, it means that the specified data can be found encoded inside the `data` block if the response was successful. If no returns are specified for a given endpoint, or in case all possible return values for an endpoint are declared as optional, `data` can just be `null`.
+The API currently contains the following endpoints. If an endpoint specifies *Returns* for either HTTP GET or HTTP POST, it means that the specified data can be found encoded inside the `data` block if the response was successful. If no Returns are specified for a given endpoint, or in case all possible return values for an endpoint are declared as optional, `data` can just be `null`.
```

Comment:
```suggestion
The API currently contains the following endpoints. If an endpoint specifies *Returns* for either HTTP GET or HTTP POST, it means that the specified data can be found encoded inside the `data` block if the response was successful. If no return values are specified for a given endpoint, or in case all possible return values for an endpoint are declared as optional, `data` can just be `null`.
```

Reply:
The reason why I chose "Returns" was that the endpoints specify *Returns*, which in this context is a proper name and not a description. I think we should keep it that way.

Reply:
I would choose to go with @katexochen's suggestion. The capitalization implies that *Returns* are some kind of special object or action intended by us (given that capitalization usually implies the name of something). However, it's just a basic JSON response just as basically any other JSON REST API. It's a fairly easy one, even.

Also, I do not think that "endpoints specify Returns" is common terminology, at least I could not find this to be commonly used with a quick Google search.

Plus highlighting \*Returns\* looks very weird: https://deploy-preview-21--edgeless-docs.netlify.app/marblerun/#/reference/coordinator

Reply:
It looks weird, indeed. Then let's change into your suggestion.